packages/vue-remark/README.md

```diff
@@ -0,0 +1,151 @@
+# @gridsome/vue-remark
+
+> Create pages with Vue components in markdown. Perfect for building Documentation, Design Systems, Portfolios, Blogs, etc.
+
+## Install
+
+- `yarn add @gridsome/vue-remark`
+- `npm install @gridsome/vue-remark`
+
+## Usage
+
+```js
+module.exports = {
+  plugins: [
+    {
+      use: '@gridsome/vue-remark',
+      options: {
+        typeName: 'MarkdownPage', // required
+        baseDir: './src/pages', // required
+      }
+    }
+  ]
+}
+```
+
+## Example component
+
+```jsx
+---
+title: A cool title
+excerpt: Lorem Ipsum is simply dummy text.
+layout: ~/layouts/Default.vue
+---
+import YouTube from '~/components/YouTube.vue'
+import data from '~/data/youtube.json'
+
+# {{ $frontmatter.title }}
+
+<YouTube :id="data.id" />
+
+> {{ $frontmatter.excerpt }}
+```
+
+## Options
+
+#### typeName
+
+- Type: `string` *required*
+
+The type name to give the pages in the GraphQL schema.
+
+#### baseDir
+
+- Type: `string` *required*
+
+The path to the directory which contains all the `.md` files. A relative path will be resolved from the project root directory.
+
+#### component
+
+- Type: `string`
```

Comment:
Would it make sense just to make this a boolean, and then if true, use templates/<typeName>, like source-filesystem?